.travis.yml

```diff
@@ -3,6 +3,26 @@ node_js:
   - "8"
   - "6"
   - "4"
+env:
+  - MONGOOSE_VERSION=5
+  - MONGOOSE_VERSION=4
```

Comment:
Is this double intended?

Reply:
yes. this creates the test matrix along with the defined node versions. see https://travis-ci.org/craftup/node-mongo-tenant